pkg/trace/api/endpoints.go

```diff
@@ -76,6 +76,12 @@ var endpoints = []endpoint{
 		Pattern: "/profiling/v1/input",
 		Handler: func(r *HTTPReceiver) http.Handler { return r.profileProxyHandler() },
 	},
+	{
+		Pattern: "/telemetry/proxy/",
+		Handler: func(r *HTTPReceiver) http.Handler {
+			return http.StripPrefix("/telemetry/proxy", r.forwardingProxyHandler(&telemetryProxyEndpointsConfig))
```

Comment:
```go
return newForwardingProxy(&proxyConfig{
	Name:        "telemetry",
    Hostname:    r.conf.Hostname,
    DefaultEnv:  r.conf.DefaultEnv,
    URLTemplate: "https://instrumentation-telemetry-intake.%s",
    DefaultSite: "datadoghq.com",
})
```
I think this makes a lot more sense. The `telemetry.go` file doesn't help. It just misplaces something that should be readily available to read here.